tools/README.md

```diff
@@ -167,3 +167,70 @@ The output chunked graph metadata will go as follows (assuming the current direc
     "edge_data": {}
 }
 ```
+
+## Change edge type to canonical edge type for partitioned graph data
```

Comment:
`graph data` seems not precise. we change the config json, right?

Reply:
That make sense, use partition config json instead